src/Raven.Server/Documents/Queries/CollectionQueryEnumerable.cs

```diff
@@ -169,7 +169,7 @@ public bool MoveNext()
                     _innerCount++;
 
                     var doc = _fieldsToFetch.IsProjection
-                        ? _resultsRetriever.GetProjectionFromDocument(_inner.Current, null, 0f, _fieldsToFetch, _context, null)
```

Comment:
where are we setting this now?

Reply:
FinishDocumentSetup

Reply:
I do not think this is the case in this flow? But I see inconsistency here, because for projection we were setting 0 and `null` otherwise. I think it should be set to 1 regardless if this is projection or not.